code/datums/martial.dm

```diff
@@ -90,6 +90,15 @@
 /datum/martial_art/proc/remove(mob/living/carbon/human/H)
 	if(H.mind.martial_art != src)
 		return
-	H.mind.martial_art = base
+	if(base)
+		H.mind.martial_art = base
+	else
+		H.mind.martial_art = H.mind.default_martial_art
 	if(help_verb)
 		H.verbs -= help_verb
+	remove_act(H)
+	H.mind.martial_art.teach(H)
+	qdel(src)
```

Comment:
There's no way src could be default_martial_art here right? Because there's only one of those ever created